docs/service_table.md

```diff
@@ -4,11 +4,13 @@ View [Service Graph](./current_architecture.md) to visualize request flows.
 
 | Service                                                      | Language      | Description                                                                                                                                  |
 |--------------------------------------------------------------|---------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| [accountingservice](./services/accountingservice.md)         | Go            | Processes incoming orders and count the sum of all orders (mock).                                                                            |
 | [adservice](./services/adservice.md)                         | Java          | Provides text ads based on given context words.                                                                                              |
 | [cartservice](./services/cartservice.md)                     | DotNet        | Stores the items in the user's shopping cart in Redis and retrieves it.                                                                      |
 | [checkoutservice](./services/checkoutservice.md)             | Go            | Retrieves user cart, prepares order and orchestrates the payment, shipping and the email notification.                                       |
 | [currencyservice](./services/currencyservice.md)             | C++           | Converts one money amount to another currency. Uses real values fetched from European Central Bank. It's the highest QPS service.            |
 | [emailservice](./services/emailservice.md)                   | Ruby          | Sends users an order confirmation email (mock).                                                                                              |
+| [frauddetectionservice](./services/frauddetectionservice.md) | Kotlin        | Analyzes incoming orders and detects fraud attempts (mock).                                                                                  |
 | [featureflagservice](./services/featureflagservice.md)       | Erlang/Elixir | CRUD feature flag service to demonstrate various scenarios like fault injection & how to emit telemetry from a feature flag reliant service. |
```

Comment:
We may need to modify the architecture map as well https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/current_architecture.md to show support for Kafka and the new services. My thought is this can be done as a separate follow on PR as this PR is quite big.

Reply:
I have added them ( 927ed69e8a013992c52163e1e68cbd9b59e82c22  &&  d701fe284353ca3310c619d757e8d9b04cc2eee8 ) but they got deleted by merging main. I will add them back in.